specs/credentialmanagement/index.src.html

```diff
@@ -821,9 +867,10 @@ <h4 id="interfaces-request-options">
 
     <pre>
       navigator.<a idl>credentials</a>.<a idl lt="get()">get</a>({
-        "<a idl>types</a>": [ "federated" ],
-        "<a idl>options</a>": {
-            "<a idl>providers</a>": [ "https://identity.example.com/" ]
+        "<a idl>filters</a>": {
+            "federated" : {
+                "<a idl>providers</a>": [ "https://identity.example.com/" ]
```

Comment:
(Basically, I think I'd suck it up and deal with the extra layer if it seemed like that would get us to a compromise folks in the CCG and WPIG were happy with. But if those groups are unhappy, and planning another API anyway, then I'd prefer to drop the additional complexity we've added entirely rather than adding more. :) )